packages/firebase_ml_vision/example/test_driver/firebase_ml_vision_test.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('firebase_ml_vision', () async {
```

Comment:
I think it's best to remove this test invocation since you're not actually testing anything here. That way it won't say the tests passed when they didn't.

I've been doing this for other plugins. Just run the three lines of the block directly.

```
    final FlutterDriver driver = await FlutterDriver.connect();
    await driver.requestData(null, timeout: const Duration(minutes: 1));
    driver.close();
```

And you can remove the package:test import above after doing this.